Android/Guava/source/Xamarin.Google.J2Objc.Annotations/Xamarin.Google.J2Objc.Annotations.targets

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <AndroidJavaLibrary Include="$(MSBuildThisFileDirectory)..\..\jar\*.jar" />
```

Comment:
Whoops, I think this line needs the `$(AndroidApplication) == true` check:

https://github.com/xamarin/AndroidX/blob/47d87a33df02bc49e753c74a26e5d26bc6556b2f/source/AndroidXTargets.cshtml#L51

Reply:
Do not worry I cannot publish anyway (testing with other community PRs). Something changed in CI paths and release pipeline cannot find signed nugets (searches in `tempsigned` and nugets are in `_XamarinComponents/nuget-signed`)